content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join-phase.md

```diff
@@ -0,0 +1,100 @@
+---
+title: kubeadm join phase
+weight: 90
+---
+<!--
+---
+title: kubeadm join phase
+weight: 90
+---
+-->
+
+<!--
+In v1.14.0, kubeadm introduces the `kubeadm join phase` command with the aim of making kubeadm more modular. This modularity enables you to invoke atomic sub-steps of the join process.
+Hence, you can let kubeadm do some parts and fill in yourself where you need customizations.
+-->
+在 v1.14.0 中 kubeadm 引入了 `kubeadm join phase` 命令，目的是使 kubeadm 更加模块化。这种模块化使你能够调用连接流程的原子步骤。
+因此，你可以让 kubeadm 完成某些部分，并在需要自定义的地方自行完成。
+
+<!--
+`kubeadm join phase` is consistent with the [kubeadm join workflow](/docs/reference/setup-tools/kubeadm/kubeadm-join/#join-workflow),
+and behind the scene both use the same code.
+-->
+`kubeadm join phase` 与 [kubeadm join 工作](/docs/reference/setup-tools/kubeadm/kubeadm-join/#join-workflow)是一致的，而在底层，两者使用相同的代码。
+
+## kubeadm join phase {#cmd-join-phase}
+
+{{< tabs name="tab-phase" >}}
+{{< tab name="phase" include="generated/kubeadm_join_phase.md" />}}
```

Comment:
looks like none of the referenced files exist in the 1.14 branch

Reply:
👍